src/prm.a51

```diff
@@ -89,6 +89,17 @@ mode2		EQU	RAMbit+9	; Mode, 2eme octet
 					; b4: 			b5: 
 					; b6: 			b7:
 
+Helpbits	EQU RAMbit+10	; If Bits are needed for Calculations
+Helpbit0	EQU (Helpbits-RAMbit)*8
+Helpbit1	EQU ((Helpbits-RAMbit)*8)+1
+Helpbit2	EQU ((Helpbits-RAMbit)*8)+2
+Helpbit3	EQU ((Helpbits-RAMbit)*8)+3
+Helpbit4	EQU ((Helpbits-RAMbit)*8)+4
+Helpbit5	EQU ((Helpbits-RAMbit)*8)+5
+Helpbit6	EQU ((Helpbits-RAMbit)*8)+6
+Helpbit7	EQU ((Helpbits-RAMbit)*8)+7
+
+
```

Comment:
yes, your understanding is correct. My problem was, that I had issues to understand which registers are already used in a subroutine. so I decided yo define special bits just because it was easier for me.
If you want I can use the B register.

Reply:
ok, changed